signac_dashboard/util.py

```diff
@@ -30,13 +30,29 @@ def ellipsis_string(string, length=60):
         return string[:half] + "..." + string[-half:]
 
 
+def ellipsis_truncate_end(string, length=60):
+    string = str(string)
+    if len(string) < length:
+        return string
+    else:
+        return string[:length] + "…"
```

Comment:
Don't use the single-character ellipsis, please.
```suggestion
        return string[:length] + "..."
```

Reply:
I don’t think this warrants using a special character. 🤷🏻‍♂️ If you need to make it one character, use the HTML entity `&hellip;`.